torchvision/transforms/v2/functional/_utils.py

```diff
@@ -169,3 +169,11 @@ def _is_cvcuda_available():
         return True
     except ImportError:
         return False
+
+
+def is_cvcuda_tensor(inpt: Any) -> bool:
```

Comment:
Can we add a docstring here? 
And
Should this be exposed in the functional/__init__.py?